src/content/blog/announcements/2024-05-31_nixos-24.05.md

```diff
@@ -0,0 +1,41 @@
+category: announcements
+---
+[![NixOS 24.05 Uakari logo](../../../assets/logo/nixos-logo-24.05-uakari-lores.png)](https://github.com/NixOS/nixos-artwork/blob/master/releases/24.05-uakari/uakari.png)
+
+Hey everyone, we are [Weijia Wang](https://github.com/wegank) and [Jörg Thalheim](https://github.com/Mic92), the release managers for this stable release and we are very proud to announce the public availability of NixOS 24.05 “Uakari”.
+
+This release will receive bugfixes and security updates for seven months (up until 2024-12-31).
+
+*   [Release manual](/manual/nixos/stable/release-notes.html#sec-release-24.05)
+    *   [Highlights](/manual/nixos/stable/release-notes.html#sec-release-24.05-highlights)
+    *   [New Services](/manual/nixos/stable/release-notes.html#sec-release-24.05-new-services)
+    *   [Backward Incompatibilities](/manual/nixos/stable/release-notes.html#sec-release-24.05-incompatibilities)
+    *   [Other Notable Changes](/manual/nixos/stable/release-notes.html#sec-release-24.05-notable-changes)
+*   [Upgrade Instructions](/manual/nixos/stable/index.html#sec-upgrading)
+*   [ISO Image Downloads](/download)
+
+The 24.05 release was made possible due to the efforts of **2491** contributors, who authored **47755** commits since the previous release.
+Our thanks go the contributors who also take care of the continued stability and security of our stable release.
+
+NixOS is already known as [the most up to date distribution](https://repology.org/repositories/statistics/newest) while also being [the distribution with the most packages](https://repology.org/repositories/statistics/total). This release saw **17775 new** packages and **16244 updated** packages in Nixpkgs. We also **removed 9393** packages in an effort to keep the package set maintainable and secure.
+In addition to packages the NixOS distribution also features modules and tests that make it what it is. This release brought **144 new** modules and **removed 23**. In that process we **added 1983** options and **removed 304**.
+
+### GNOME 46
+
+GNOME has been updated to version 46 "Kathmandu", which introduces an enhanced Files app, support for OneDrive in Online Accounts, and more changes. Refer to the [release notes](https://release.gnome.org/46/) for more details.
+
+### Plasma 6
+
+Plasma has been updated to version 6.0, which introduces new defaults, refreshed Breeze, and more changes. Refer to the [release notes](https://kde.org/announcements/megarelease/6/) for more details.
+
+### Special Thanks
+
+We want to personally thank [Jacek Generowicz](https://github.com/jacg) for editorializing the release notes, [Vladimír Čunát](https://github.com/vcunat) for his tireless effort in managing jobsets, staging cycles and build infrastructure, [Cole Helbling](https://github.com/cole-h) for his epic effort in bisecting kernel issues with ARM64 for our remote builders, and [Graham Christensen](https://github.com/grahamc) for dutifully tending to our build infrastructure.
```

Comment:
Does Graham still manage the NixOS build infrastructure? I think Hexa is doing now a fair part there. I don't know who else is involved.